pkg/controllers/membercluster/v1beta1/membercluster_controller.go

```diff
@@ -371,19 +371,19 @@ func (r *Reconciler) syncRoleBinding(ctx context.Context, mc *clusterv1beta1.Mem
 	}
 	currentRoleBinding.Subjects = expectedRoleBinding.Subjects
 	currentRoleBinding.RoleRef = expectedRoleBinding.RoleRef
-	klog.V(2).InfoS("updating role binding", "memberCluster", klog.KObj(mc), "roleBinding", roleBindingName)
```

Comment:
Why are we remove the roleBinding key here ?

Reply:
The key name is derived from the mc name, I guess it doesn't hurt to put it back but it's kinda redundant.